src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -15,18 +15,23 @@
     left: 50%;
   }
 
-  padding: 15px;
-  border-radius: 0 0 50px 50px;
+  padding: $announcement-container-padding;
```

Comment:
If this padding is specifically just for this file (unlikely to be shared elsewhere) it’s fine for it to be defined at the top of the file, rather than in the ‘global’ constants file